api/helpers/gamestate/moves/resolve/execute.js

```diff
@@ -71,10 +71,21 @@ module.exports = {
           oneOff, // oneOff stays on the stack until next move
           turn: result.turn - 1, // turn doesn't increment until the next move
```

Comment:
This isn't needed now that we removed the moved the code that increments the turn from line 49 to line 87. We should remove this so that the turn stays the same for this block